pyproject.toml

```diff
@@ -1,41 +1,25 @@
 [build-system]
-requires = ["setuptools >= 75.3.0", "wheel"]
+requires = ["setuptools >= 72.1.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
-[tool.setuptools]
-include-package-data = false
-
-[tool.setuptools.packages.find]
-include = ["yoyodyne*"]
-exclude = ["examples*"]
-
 [project]
 name = "yoyodyne"
-version = "0.3.3"
+version = "0.4.0"
 description = "Small-vocabulary neural sequence-to-sequence models"
-readme = "README.md"
-requires-python = ">= 3.10"
 license = { text = "Apache 2.0" }
+readme = "README.md"
+requires-python = ">= 3.10,<3.141"
 authors = [
-    {name = "Adam Wiemerslage"},
-    {name = "Kyle Gorman"},
-    {name = "Travis Bartley"},
+    { name = "Adam Wiemerslage" },
+    { name = "Kyle Gorman", email = "kylebgorman@gmail.com" },
+    { name = "Travis Bartley" },
```

Comment:
small vanglorious thing, i use my middle initial M.

Reply:
Can you submit a 1-line PR and then I'll merge in?